src/test/e2e/28_temp_directory_deploy_test.go

```diff
@@ -0,0 +1,53 @@
+package test
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/defenseunicorns/zarf/src/internal/k8s"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTempDirectoryDeploy(t *testing.T) {
+	t.Log("E2E: Temporary directory deploy")
+
+	// run `zarf package deploy` with a specified tmp location
+	otherTmpPath := "/tmp/othertmp"
+
+	e2e.cleanFiles(otherTmpPath)
+
+	e2e.setup(t)
+	defer e2e.teardown(t)
+
+	path := fmt.Sprintf("build/zarf-package-dos-games-%s.tar.zst", e2e.arch)
+
+	_ = os.Mkdir(otherTmpPath, 0750)
+
+	// Deploy the game
+	stdOut, stdErr, err := e2e.execZarfCommand("package", "deploy", path, "--confirm", "--tmpdir", otherTmpPath, "--log-level=debug")
+	require.Contains(t, stdErr, otherTmpPath, "The other tmp path should show as being created")
+	require.NoError(t, err, stdOut, stdErr)
+
+	// Get a random local port for this instance
+	localPort, _ := k8s.GetAvailablePort()
```

Comment:
Same as above, I'd look at using the same logic as `01_component_choice_test.go`, only without testing the variables themselves as they wouldn't matter for this test.

Reply:
👍